vector/src/main/java/im/vector/app/core/ui/views/BottomSheetActionButton.kt

```diff
@@ -81,6 +81,7 @@ class BottomSheetActionButton @JvmOverloads constructor(
         set(value) {
             field = value
             views.bottomSheetActionIcon.setImageDrawable(value)
+            views.bottomSheetActionIcon.isVisible = field != null
```

Comment:
We would need an extension `setImageDrawableOrHide`

Reply:
Done